tests/taskcluster/tasks/raptor.yml

```diff
@@ -34,3 +37,9 @@ task:
         type: directory
         path: '/home/tester/logs'
         expires: '{{nextYear}}'
+
+  extra:
+    treeherder:
+      groupSymbol: Raptor
```

Comment:
I've changed the group name to 'Raptor' but for some reason for me, on the PR tests on treeherder it still appears in a group named 'Rpt'. Not sure why.